libdevice/include/device-sanitizer-report.hpp

```diff
@@ -12,10 +12,11 @@
 
 enum class DeviceSanitizerErrorType : int32_t {
   UNKNOWN,
-  OUT_OF_BOUND,
+  OUT_OF_BOUNDS,
   MISALIGNED,
   USE_AFTER_FREE,
-  OUT_OF_SHADOW_BOUND,
+  OUT_OF_SHADOW_BOUNDS,
```

Comment:
IIRC OUT_OF_SHADOW_BOUNDS will be removed per our offline discussion?

Reply:
I think currently it's useful, because DeviceSanitizer is not stable enough, it can help to catch some unexpected error. Maybe, when we support check private memory, we will not need it.